examples/ale/train_a3c_ale.py

```diff
@@ -88,11 +89,18 @@ def main():
     parser.set_defaults(use_lstm=False)
     args = parser.parse_args()
 
-    if args.seed is not None:
-        misc.set_random_seed(args.seed)
+    # Set a random seed used in ChainerRL.
+    # If you use more than one processes, the results will be no longer
+    # deterministic even with the same random seed.
+    misc.set_random_seed(args.seed)
 
-    args.outdir = experiments.prepare_output_dir(args, args.outdir)
+    # Set different random seeds for different subprocesses.
+    # If seed=0 and processes=4, subprocess seeds are [0, 1, 2, 3].
+    # If seed=1 and processes=4, subprocess seeds are [4, 5, 6, 7].
+    assert (args.seed + 1) * args.processes < 2 ** 31
```

Comment:
The check would be `assert (args.seed + 1) * args.processes - 1 < 2 ** 31`.  It might be clearer to check it by `process_seeds.max() < 2 ** 31`.